VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre-menu/command-centre-menu.component.html

```diff
@@ -1,8 +1,11 @@
 <div class="tabcontainer">
   <div class="tab tab--left">
-    <button id="hearingsTabButton" [ngClass]="{ active: currentMenu === 'Hearing' }" (click)="displayHearing()">Hearings</button>
+      <button id="hearingsTabButton" [ngClass]="{ active: currentMenu === 'Hearing' }" (click)="displayHearing()">Hearings</button>
   </div>
-  <div class="tab tab--right">
-    <button id="messagesTabButton" [ngClass]="{ active: currentMenu === 'Message' }" (click)="displayMessages()">Messages</button>
+  <div [ngClass]="isStaffMember ? 'tab tab--middle' : 'tab tab--right'">
+      <button id="messagesTabButton" [ngClass]="{ active: currentMenu === 'Message' }" (click)="displayMessages()">Messages</button>
+  </div>
+  <div  class="tab tab--right" *ngIf="isStaffMember">
+      <button id="start-hearing-btn" (click)="signIntoConference()" [disabled]="!conferenceId" > Join Hearing </button>
```

Comment:
Not a deal breaker but after bearing some of the pain of the UI testing, could we start making the ids a bit more unique so it we know it's a start-hearing-btn for the command centre

Reply:
will do